src/server/redis.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import { promisify } from 'util'
+import redis from 'redis'
+import redisMock from 'redis-mock'
+
+const client =
+  process.env.NODE_ENV === 'test'
+    ? redisMock.createClient()
```

Comment:
During tests we don’t want to impact our local redis server so we simply run on an in-memory mock.